python_modules/dagster/dagster/core/storage/file_manager.py

```diff
@@ -158,10 +158,13 @@ def write_data(self, data: bytes, ext: str = None) -> FileHandle:
         raise NotImplementedError()
 
 
-@resource(config_schema={"base_dir": Field(StringSource, default_value=".", is_required=False)})
+@resource(config_schema={"base_dir": Field(StringSource, is_required=False)})
 def local_file_manager(init_context):
     """FileManager that provides abstract access to a local filesystem.
 
+    By default, files will be stored in the directory specified by local_artifact_storage in
+    your dagster.yaml file (which will be a temporary directory if not explicitly set).
+
```

Comment:
I think this reads a bit confusing, which is fair since the behavior its trying to describe is non trivial. Maybe just say it defaults to `$DAGSTER_HOME/storage/file_manager` and avoid explaining the local_artifact_storage override or $DAGSTER_HOME temp-dir fallback stuff.

Reply:
`$DAGSTER_HOME/storage/file_manager` will be the default but it's not always accurate - if a user sets local_artifact_storage, the default base dir for file manager will be `<local_artifact_storage>/storage/file_manager`.

Reply:
Yea it just feels excessive to explain that in detail here - do we mention that in the other IO managers?

Maybe just something "unless changed via instance config" could get it across.

I think you can drop the temporary director piece since that behavior only happens in `dagit` and there is a message when that happens about what is going on.

Reply:
I dont feel that strongly here so up to you

Reply:
yes we mention those details in IO Managers. i updated it to not mention the temp dir piece.